src/styles/utils/_vars.scss

```
@@ -1 +1,28 @@
-$c-gray: #eee;
+$mainBackground: #f2f2f2;
+
+$white: #fff;
+$black: #000;
+$blue: #0075ff;
+$liteGray: #9fadc2;
+$textColor: #3e3e3e;
+$inputBackground: #ebecf1;
+$placeholderBacground: #b9c1ce;
+$mainFontFamily: "Roboto", sans-serif, arial;
```

чтобы было понятно что за фонт
```suggestion
$robotoFontFamily: "Roboto", sans-serif, arial;
```

---

А если заказчик захочет поменять главный шрифт с Roboto на Аrial мне потом все переменные в коне менять?